Gems/AudioSystem/Code/Source/Engine/ATLEntities.cpp

```diff
@@ -285,5 +286,22 @@ namespace Audio
 
         return sResult;
     }
+    CATLAudioFileEntry::CATLAudioFileEntry(const char * const filePath, IATLAudioFileEntryData * const implData)
+        : m_filePath(filePath)
+        , m_fileSize(0)
+        , m_useCount(0)
+        , m_memoryBlockAlignment(AUDIO_MEMORY_ALIGNMENT)
+        , m_flags(eAFF_NOTFOUND)
+        , m_dataScope(eADS_ALL)
+        , m_memoryBlock(nullptr)
+        , m_implData(implData)
+    {
+    }
+
+    CATLAudioFileEntry::~CATLAudioFileEntry()
+    {
+
+    }
```

Comment:
FYI you can use default here as well in a cpp file
`CATLEAudioFileEntry::~CATLEAudioFileEntry() = default;`